pages/contribute.md

```diff
@@ -3,24 +3,119 @@ layout: default
+## Overview of how you can contribute
 
-1. To join the Auto-WCAG Community, you need a W3C Account.
-   If you don't already have one, you can get it here:
-   [https://www.w3.org/accounts/request].
-2. Go to **[Join the Auto-WcAG community group now!](https://www.w3.org/community/wp-login.php?redirect_to=%2Fcommunity%2Fauto-wcag%2Fjoin)** and join the group
-3. If you work for a W3C Member organization, you will need to request approval.
-   This request will be sent to your organization's representative in the W3C's
-   Advisory Committee.
+* [Give feedback on rules](#Give-feedback-on-rules)
+    * [Feedback for ideas and early drafts (Issues on Github)](#)
+    * [Feedback and reviews for draft rules (Pull Requests on Github)](#Feedback-for-ideas-and-early-drafts-(Issues-on-Github))
+    * [Feedback/corrections for published rules](#Feedback/corrections-for-published-rules)
+* [Suggest new rules](#Suggest-new-rules)
+    * [Submit an idea for discussion (Issue on Github)](#Submit-an-idea-for-discussion-(Issue-on-Github))
+    * [Submit a draft for a rule (Pull Request on Github)](#Submit-a-draft-for-a-rule-(Pull-Request-on-Github)
+* [Join the Auto-WCAG Community Group](#Join-the-Auto-WCAG-Community-Group)
+    * [Mailing list keeps you up to date](#Mailing-list-keeps-you-up-to-date)
+    * [Participate in our monthly calls](#Participate-in our-monthly-calls)
+    * [How to join the Auto-WCAG Community Group](#How-to-join-the-Auto-WCAG-Community-Group)
 
-## Participate in our monthly calls
+## Give feedback on rules
 
-Auto-WCAG has conference calls every month (usually) on the second Thursdays at 16:00 Central European Time. Participating in these calls is the easiest way to become an active contributor. The invitation and agenda for these meetings is sent out through the mailing list a few days in advance.
+Anyone can provide feedback on rules on our open [Auto-WCAG Github repository](https://github.com/auto-wcag/auto-wcag), so you don't even have to be a member of the Auto-WCAG Community Group to contribute in this way.
+
+### Feedback for ideas and early drafts (Issues on Github) 
+Rules and Algorithms (shared terms across rules) that are Github Issues can be everything from just a title to a quite finished rule. The issues will be prefixed with either "Rule:" (or possibly "New Rule:") or "Algorithm". 
+
+The top comment on an issue should hold the updated rule or algorithm, and underneath there will be a thread of comments and a history of actions related to the issue. 
+
+Please feel free to add your own comments to issues.
+
+Start giving feedback on issues:
+
+- See the [ideas and early drafts that might need input](https://github.com/auto-wcag/auto-wcag/issues) 
+
+- Use the [checklist for reviewers](#) (TBD) to guide your review
+- Learn more about [commenting on issues](#) (TBD)
```

Comment:
I think we cannot merge this with TBD links. Why are these TBD? Can I help to define the structure so that we can make them meaningful links?

Reply:
Because we haven't written the guiding material...